.github/workflows/release-snapshot.yml

```diff
@@ -0,0 +1,29 @@
+name: Release Snapshot to Sonatype
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release_snapshot_sonatype:
+    if: "github.event.release.prerelease"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: corretto
+          java-version: 8
+          cache: sbt
+      - name: Release Snapshot to Sonatype
+        run: |
+          VERSION=$(git describe --tags | cut -f2 -d"@")
```

Comment:
We're expecting a snapshot release to named with specific syntax:
`X.X.X-SNAPSHOT`

Note the missing `v` from the beginning of the release title, and the mandatory `-SNAPSHOT` at the end. Probably there'd need to be extra instructions in the readme about how to handle this

Reply:
Any chance that we could tolerate the "v" on the start by stripping it off here? Just thinking that it's likely to be an obscure failure